internal/backend/temporalclient/client.go

```diff
@@ -76,11 +81,19 @@ func New(cfg *Config, z *zap.Logger) (Client, error) {
 	}, nil
 }
 
-func (c *impl) startDevServer(ctx context.Context, cfg *Config, opts client.Options) error {
-	cfg.DevServer.ClientOptions = &opts
-
+func (c *impl) startDevServer(ctx context.Context) error {
 	var err error
-	if c.srv, err = testsuite.StartDevServer(ctx, cfg.DevServer); err != nil {
+	logPath := path.Join(xdg.DataHomeDir(), "temporal_dev.log")
+	c.log, err = os.OpenFile(logPath, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0o644)
```

Comment:
this can accumulate pretty quickly, i would drop the append.

Reply:
I thought `O_WRONLY` forces truncation anyway, but you're right, so dropping `O_APPEND`.

See verification here: https://goplay.tools/snippet/lKfAXSidc97